### ISSUE TITLE
PhaseOne 645DF/P65+ noise profile

### ISSUE THREAD

**Issue by dim162** [darktable-org/darktable#3706]
Hello!
I create noise profile for the PhaseOne 645DF/P65+. It's done with the quadrant problem. I give the icm used in CaptureOne as white balanced presets, & also an ICC homemade with Argyll-cms.
[darktable-noiseprofile-20181231.tar.gz](https://github.com/darktable-org/darktable/files/3937134/darktable-noiseprofile-20181231.tar.gz)

[PhaseOneP65+icm&icc.tar.gz](https://github.com/darktable-org/darktable/files/3937136/PhaseOneP65%2Bicm.icc.tar.gz)

**Comment by rawfiner** [comment]
The graphs of the noise profile look strange to me. Please follow this tutorial to take the pictures, and keep the thumbnails in the tar.gz so that I can check them: https://pixls.us/articles/how-to-create-camera-noise-profiles-for-darktable/
Thanks

**Comment by dim162** [comment]
I was not connected for a few days, sorry! I have followed this tutorial. I
can't find the folder with the thumbnails... I gonna redo the procedure.
Don't forget, phaseone is a ccd & 16 bits raw file

**Comment by dim162** [comment]
I found it Dropbox images-noise
<https://www.dropbox.com/sh/do6rau2i3b4xjj6/AAAkq4qvsyXeriZ4Hnh74m2_a?dl=0>

Le lun. 6 janv. 2020 à 18:39, Dimitri Gathy <dimitri.gathy@gmail.com> a
écrit :

**Comment by rawfiner** [comment]
On your images, the highlight part looks quite sharp, maybe that is what gives the strange profiles, could you maybe try to defocus more?

**Comment by dim162** [comment]
I cannot do the noise profile, the "darktable-gen-noiseprofile" send an
error message (i copy the error message in a txt file). The pictures were
done on the front cover of a little softbox & a 500j studio-flash set at
75%, the camera is mounted with a f2.8 80mm at f11 & i shoot at 1/1600s,
the lens is at 50cm from the gradient & set to infinity. When i opened the
images in Dt, it's under & over exposed by a minus of 2 stops. I changed
the ISO vs the flash power, all in manual. The whole set with the error
message.
<https://www.dropbox.com/sh/b9e2y0352ob71z1/AAB095gG21WpskbRGioRnkSHa?dl=0>

**Comment by dim162** [comment]
There is still a an issue #207
<https://github.com/darktable-org/rawspeed/issues/207> with the Phase1 P65+

Le ven. 17 janv. 2020 à 12:37, Dimitri Gathy <dimitri.gathy@gmail.com> a
écrit :

**Comment by rawfiner** [comment]
The important line in the error message is this one, for iso 800:
$"/home/dim/Desktop/phase1noise/Capture28596.IIQ" not over-exposed (4)$
That's what makes the script stop

**Comment by dim162** [comment]
I have the same message for the images with the smalest ISO settings. The
difference is the message: "The following ISO settings have incorrect
images". I open the images in Dt with no modules switched on & just basic
modules (input color profile: linear Rec2020 RGB, output color profile,
demosaic, raw b/w point:128-65535), the histograms are nearly the same, the
difference are the noise, the levels are the same, only 5 strips are well
exposed, 1 strips are overexposed, the rest is underexposed. I shoot in a
totally black surrounding, no light reflections, at f11 & 1/1600 sec (it's
a lens shutter), i can't do more!

**Comment by dim162** [comment]
I also retried with the first set of image i sent you which succeded, the
same error message

Le mer. 22 janv. 2020 à 13:22, Dimitri Gathy <dimitri.gathy@gmail.com> a
écrit :

**Comment by rawfiner** [comment]
I would prefer a much wider aperture than f11 to get a smoother blur, but still you can try to generate the noiseprofiles even if the script complains using the "-C" option.

**Comment by dim162** [comment]
It's a full frame middle format camera, in full frame 24x36, it corresponds
to f:5,6 depth of field, but i do

**Comment by dim162** [comment]
With "-C", the script goes by but nothing is saved

Le jeu. 23 janv. 2020 à 07:42, Dimitri Gathy <dimitri.gathy@gmail.com> a
écrit :

**Comment by rawfiner** [comment]
I tried running the noiseprofile generation myself with the images, but unfortunately the resulting profile is not good sorry...
Here it is (rename it to presets.json) if you want to use it for yourself, but I can't integrate it into darktable, profile is bad for ISO 50 and 100 and okish for ISO 200.
[presets.txt](https://github.com/darktable-org/darktable/files/7875430/presets.txt)
[noise_result.pdf](https://github.com/darktable-org/darktable/files/7875427/noise_result.pdf)

**Comment by dim162** [comment]
PhaseOne P65+ are not (well) supported in rawspeed.
https://github.com/darktable-org/rawspeed/issues/207

**Comment by rawfiner** [comment]
thanks for the info @dim162 that may explain the bad profile quality!

**Comment by dim162** [comment]
Of course, not immediately, i have no machine free & free time now to test
it
